Tests involving Strings fail when run on Windows

When running Gaffer tests on Windows 10, tests which involve reading `String` data from disk and comparing it to those not from disk fail.
This is an example from the `serialisation` tests - [code here](https://github.com/gchq/Gaffer/blob/eec0c3f1e9851e1310b7345b1fe97d812c497608/core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/implementation/MultiSerialiserTest.java#L68):
```
[ERROR] Failures:
[ERROR]   MultiSerialiserTest.shouldMatchHistoricalFileSerialisation:80 expected: <{
  "serialisers" : [ {
    "key" : 0,
    "serialiser" : {
      "class" : "uk.gov.gchq.gaffer.serialisation.implementation.StringSerialiser"
    },
    "valueClass" : "java.lang.String"
  }, {
    "key" : 1,
    "serialiser" : {
      "class" : "uk.gov.gchq.gaffer.serialisation.implementation.raw.CompactRawLongSerialiser"
    },
    "valueClass" : "java.lang.Long"
  }, {
    "key" : 2,
    "serialiser" : {
      "class" : "uk.gov.gchq.gaffer.serialisation.implementation.raw.CompactRawIntegerSerialiser"
    },
    "valueClass" : "java.lang.Integer"
  } ]
}> but was: <{
  "serialisers" : [ {
    "key" : 0,
    "serialiser" : {
      "class" : "uk.gov.gchq.gaffer.serialisation.implementation.StringSerialiser"
    },
    "valueClass" : "java.lang.String"
  }, {
    "key" : 1,
    "serialiser" : {
      "class" : "uk.gov.gchq.gaffer.serialisation.implementation.raw.CompactRawLongSerialiser"
    },
    "valueClass" : "java.lang.Long"
  }, {
    "key" : 2,
    "serialiser" : {
      "class" : "uk.gov.gchq.gaffer.serialisation.implementation.raw.CompactRawIntegerSerialiser"
    },
    "valueClass" : "java.lang.Integer"
  } ]
}>
```
In the terminal, the two JSON strings are visibly the same. The reason for the failure will be the encoding used for the file on disk, or more likely the line endings. This problem can hopefully be corrected by ensuring that UTF-8 is always used (not some other system default) and that Unix line endings are always used and not the system default (only Windows doesn't use Unix line endings).

--- a/core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/implementation/MultiSerialiserTest.java
+++ b/core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/implementation/MultiSerialiserTest.java
@@ -68,7 +68,7 @@ public class MultiSerialiserTest extends ToBytesSerialisationTest<Object> {
     public void shouldMatchHistoricalFileSerialisation() throws IOException, GafferCheckedException {
         final String fromDisk = IOUtils.readLines(StreamUtil.openStream(getClass(), PATH))
                 .stream()
-                .collect(Collectors.joining("\n"));
+                .collect(Collectors.joining(System.lineSeparator()));
 
         final MultiSerialiser multiSerialiser = new MultiSerialiser()
                 .addSerialiser((byte) 0, new StringSerialiser(), String.class)

--- a/core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/implementation/MultiSerialiserTest.java
+++ b/core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/implementation/MultiSerialiserTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018-2020 Crown Copyright
+ * Copyright 2018-2022 Crown Copyright
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package uk.gov.gchq.gaffer.serialisation.implementation;
 
 import org.apache.commons.io.IOUtils;

